build/root/.bazelrc

```diff
@@ -5,6 +5,9 @@ startup --expand_configs_in_place
 # https://github.com/bazelbuild/bazel/issues/7899
 build --host_force_python=PY2
 
+# https://github.com/kubernetes/kubernetes/issues/80763
```

Comment:
filed https://github.com/kubernetes/kubernetes/issues/80763 to track and added a comment here referencing it